Library/Tracking/KSRAnalytics.swift

```diff
@@ -1374,6 +1374,23 @@ public final class KSRAnalytics {
     )
   }
 
+  // MARK: - Manage Pledge Events
```

Comment:
Can you remove this MARK and move the method up into the section called `Pledge Events`